config.yml

```diff
@@ -0,0 +1,132 @@
+
+      # There might be different accounts and access token for different
+      # integrations. Better to ask for the ENV VAR NAME instead of
+      # enforcing the var name?
+      account_id: ERWIN_BB_ACCOUNT_ID
+      access_token: ERWIN_BB_ACCESS_TOKEN
+
+      # 'kind' format varies depending on type: Bitbucket or Github
+      # Bitbucket supports 'project/<name>'' and 'user/<name>''
+      # Github supports 'org/<name>' and 'user/<name>'
+      kind: project/SNK
+      repos:
+        # List of filters to be evaluated in order.
+        # Order does not matter other than includes are processed
+        # before excludes. Regexes implied by leading and trailing '/'
+        filters:
+          include:
+            - python-utility
+            - /^.*-wrapper$/
+          exclude:
+            - java-wrapper
+            - /^.*-legacy$/
+    target:
+      type: github-enterprise
+      base_url: https://erw-github.company.com
+      account_id: ERW_GHE_ACCOUNT_ID
+      access_token: ERW_GHE_ACCESS_TOKEN
+
+      kind: org/snk
+      # teams are list of teams to add (no pattern support)
+      teams:
+        read_only:
+          - team-alpha
+        read_write:
+          - team-beta
+          - team-gamma
+      users:
+        read_only:
+          - user-id-1
+        read_write:
+          - user-id-2
+      # branch_modifiers allow you to rewrite target repo branch names
+      # By default, empty branch_modifiers sync exactly the same branch
+      # names from source to target
+      branch_modifiers:
+        - name: master-branch-only
+          # match can accept regex implied by leading and trailing '/'
+          match: master
+          prefix: bb-
+      branch_protection:
+        - name: protection-for-master
+          match: master
+          # The required payload for these protection requests have
+          # multiple nested fields. Yet to figure out a config format
+          protections:
+            - required_pull_request_reviews
+            - required_status_checks
+      # Example (no need to implement):
+      # - name: all-branches-except-master
+      #   match: /^(?!master).*/
+      #   prefix: bb-
+      # Example (no need to implement):
+      # - name: all-branches
+      #   match: /.*/
+      #   prefix: bb-
+      # Example (no need to implement):
+      # - name: rename-branch
+      #   match: featureX
+      #   rename: featureY
+
+  ## Integration 2
+  - name: personal-roger-bb-to-ghe
+    enabled: false
+    sync:
+      type: once
+      period_seconds: 0
+    source:
+      type: bitbucket-cloud
+      base_url: https://bitbucket.org/roger
+      account_id: ROGER_BB_ACCOUNT_ID
+      access_token: ROGER_BB_ACCESS_TOKEN
+      kind: user/roger
+      repos:
+        filters:
+          include:
+            - /.*/
+          exclude: []
+    target:
+      type: github-enterprise
+      base_url: https://rog-github.company.com
+      account_id: ROGER_GHE_ACCOUNT_ID
```

Comment:
Is this an ENV var or supposed to be replaced by the real secret?

If to be replaced, surround with angle brackets or something to show it, and a comment to explain.

```
<REPLACE_ME>
```